_monitoring-your-cluster/pa/index.md

```diff
@@ -292,13 +305,12 @@ The following is an example response:
 }
 ```
 
-## Performance analyzer and RCA API references
 
 ### Related links
 
-Further documentation on the use of performance analyzer and RCA can be found at the following links:
+Further documentation on the use of Performance Analyzer and RCA can be found at the following links:
 
-- [Performance analyzer API guide]({{site.url}}{{site.baseurl}}/monitoring-plugins/pa/api/).
-- [RCA]({{site.url}}{{site.baseurl}}/monitoring-plugins/pa/rca/index/).
-- [RCA API guide]({{site.url}}{{site.baseurl}}/monitoring-plugins/pa/rca/api/).
-- [RFC: Root cause analysis](https://github.com/opensearch-project/performance-analyzer-rca/blob/main/docs/rfc-rca.pdf).
\ No newline at end of file
+- [Performance Analyzer API]({{site.url}}{{site.baseurl}}/monitoring-your-cluster/pa/api/).
```

Comment:
No periods necessary here because the list items aren't complete sentences and don't complete the introductory stem.